src/providers/historical-telemetry-provider.js

```diff
@@ -49,7 +59,11 @@ export default class YamcsHistoricalTelemetryProvider {
             size = 1000;
         }
 
-        let url = this.url + 'api/archive/' + this.instance + '/parameters' + idToQualifiedName(id);
+        let url = this.url + 'api/archive/' + this.instance;
+        url += id === OBJECT_TYPES.EVENTS_OBJECT_TYPE
+            ? '/' + idToQualifiedName(id)
+            : '/parameters' + idToQualifiedName(id);
+
```

Comment:
Can we break this into an if and else block? I dont like the idea of using ternaries to differentiate urls for diff types

Reply:
Even better would be a helper function to get url, which takes care of the logic, which would make it easier to extend in the future